core/chains/evm/logpoller/parser.go

```diff
@@ -26,6 +27,10 @@ const (
 
 var (
 	ErrUnexpectedCursorFormat = errors.New("unexpected cursor format")
+	logsFields                = [...]string{"evm_chain_id", "log_index", "block_hash", "block_number",
```

Comment:
This field is returned to all of the products using LogPoller in the `logpoller.Log` response from all queries:
```
// Log represents an EVM log.
type Log struct {
    EvmChainId     *big.Big
    LogIndex       int64
    BlockHash      common.Hash
    BlockNumber    int64
    BlockTimestamp time.Time
    Topics         pq.ByteaArray
    EventSig       common.Hash
    Address        common.Address
    TxHash         common.Hash
    Data           []byte
    CreatedAt      time.Time
}

// LogPollerBlock represents an unfinalized block
// used for reorg detection when polling.
type LogPollerBlock struct {
    EvmChainId *big.Big
    BlockHash  common.Hash
    // Note geth uses int64 internally https://github.com/ethereum/go-ethereum/blob/f66f1a16b3c480d3a43ac7e8a09ab3e362e96ae4/eth/filters/api.go#L340
    BlockNumber          int64
    BlockTimestamp       time.Time
    FinalizedBlockNumber int64
    CreatedAt            time.Time
}
```
I don't know if any of them actually care about it, but we can't just return the `CreatedAt` field as 0 for everything as that would be seen as a bug.  I think there's a good case to be made that we could probably just remove this field entirely, but because that's a user-facing change I wouldn't want to do that without checking with product teams and surveying the codebase. (Also, I'd probably want to either remove it for both `Logs` and `LogPollerBlock` or neither, to keep things more consistent.)  I'd say that's out of scope for this PR, but I can create a JIRA for it.

Reply:
Makes sense, creating a JIRA item in the backlog to rethink this field is good enough ;)